test/e2e/storage/persistent_volumes-local.go

```diff
@@ -716,10 +716,8 @@ func deletePodAndPVCs(config *localTestConfig, pod *v1.Pod) error {
 
 type makeLocalPodWith func(config *localTestConfig, volume *localTestVolume, nodeName string) *v1.Pod
 
-func testPodWithNodeConflict(config *localTestConfig, testVolType localVolumeType, nodeName string, makeLocalPodFunc makeLocalPodWith, bindingMode storagev1.VolumeBindingMode) {
-	ginkgo.By(fmt.Sprintf("local-volume-type: %s", testVolType))
-	testVols := setupLocalVolumesPVCsPVs(config, testVolType, config.randomNode, 1, bindingMode)
-	testVol := testVols[0]
```

Comment:
This makes sense to me: we don't need to call `setupLocalVolumesPVCsPVs()` again here because it was already called at `BeforeEach()` above. Plus the volumes created in this line are never clean up.

Apparently we could have removed this line we [introduced setupLocalVolumesPVCsPVs() in BeforeEach()](https://github.com/kubernetes/kubernetes/commit/95f04eebeebb685318232a2e6d1360e08b58a50e).

Reply:
> /lgtm
> 
> CC @msau42 @gnufied for validation

Oh, just noticed that this PR was already LGTM'ed, sorry.